worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+    display_name = "Progressive Kazooie Shoes"
+
+class ProgressiveSwimming(Choice):
+    """Set to Basic: Dive to Double Air to Faster Swimming.
+    Set to Advanced: Dive to Sub Aqua Aiming to Talon Torpedo to Double Air to Faster Swimming.
+    Randomize Moves and Randomize BK Moves are required."""
+    display_name = "Progressive Water Training"
+    option_none = 0
+    option_basic = 1
+    option_advanced = 2
+
+class ProgressiveEggAim(Choice):
+    """Set to Basic: Third Person Egg Shooting to Egg Aim. 
+    Set to Advanced: Third Person Egg Shooting to Amaze-O-Gaze to Egg Aim to Breegull Blaster.
+    Randomize Moves and Randomize BK Moves are required."""
+    display_name = "Progressive Egg Aim"
+    option_none = 0
+    option_basic = 1
+    option_advanced = 2
+
+class ProgressiveFlight(Toggle):
+    """Flight Pad to Beak Bomb to Airborne Egg Aim. Randomize Moves and Randomize BK Moves are required."""
+    display_name = "Progressive Flight"
+
+class ProgressiveBashAttack(Toggle):
+    """Ground Rat-a-tat Rap to Breegull Bash. Randomize Stop N Swap and Randomize BK Moves are required"""
+    display_name = "Progressive Bash Attack"
+
+class EnableCheatoRewards(DefaultOnToggle):
+    """Cheato rewards you with a cheat and an additional randomized reward. Use Cheato Pages as Filler cannot be set if this is enabled."""
+    display_name = "Cheato Rewards"
+
+class EnableMultiWorldJinjos(DefaultOnToggle):
+    """Jinjos have fled to other worlds. Other players need to return them home."""
+    display_name = "Randomize Jinjos"
+
+class EnableMultiWorldDoubloons(Toggle):
+    """Jolly Roger's Doubloons are scattered across the MultiWorld."""
+    display_name = "Randomize Doubloons"
+
+class EnableMultiWorldCheatoPages(DefaultOnToggle):
+    """Cheato pages are scattered across the MultiWorld."""
+    display_name = "Randomize Cheato Pages"
+
+class EnableMultiWorldHoneycombs(DefaultOnToggle):
+    """Honeycombs are scattered across the MultiWorld."""
+    display_name = "Randomize Honeycombs"
+
+class EnableHoneyBRewards(DefaultOnToggle):
+    """Honey B gives you health and an additional randomized reward."""
+    display_name = "Honey B Rewards"
+
+class EnableMultiWorldGlowbos(DefaultOnToggle):
+    """Glowbos are scattered across the MultiWorld."""
+    display_name = "Randomize Glowbos"
+
+class EnableMultiWorldTrebleClefs(DefaultOnToggle):
+    """Treble Clefs are scattered across the MultiWorld."""
+    display_name = "Randomize Treble Clefs"
+
+class EnableMultiWorldTrainStationSwitches(Toggle):
+    """Train Stations are scattered across the MultiWorld."""
+    display_name = "Randomize Train Station Switches"
+
+class EnableMultiWorldChuffyTrain(Toggle):
+    """Chuffy is lost somewhere in the MultiWorld. 
+    Once received, you can call Chuffy at any unlocked station without defeating Old King Coal."""
+    display_name = "Chuffy as a randomized AP Item"
```

Comment:
```suggestion
    display_name = "Chuffy as a Randomized AP Item"
```
Other options are in title case, so this should be too